sdk/lib/_internal/js_runtime/lib/js_mirrors.dart

```diff
@@ -475,13 +472,12 @@ class JsLibraryMirror extends JsDeclarationMirror with JsObjectMirror
       }
       bool isConstructor = unmangledName.startsWith('new ');
       bool isStatic = !isConstructor; // Top-level functions are static, but
-                                      // constructors are not.
+      // constructors are not.
```

Comment:
Could you please fix this comment as well.
I would just put it above the `bool isStatic = !isConstructor;` line.